swan-lake/resources/featured-scenarios/create-your-first-connector-with-ballerina.md

```diff
@@ -0,0 +1,176 @@
+---
+layout: ballerina-create-your-first-connector-left-nav-pages-swanlake
+title: Create your first connector with Ballerina
+description: step-by-step guide to create Ballerina connectors using OpenAPI specifications.
+keywords: ballerina, connector, library, REST, API, openAPI, Ballerina Central
+permalink: /learn/create-your-first-connector-with-ballerina/
+active: create-your-first-connector-with-ballerina
+intro: This guide walks you through creating your first Ballerina connector using an OpenAPI specification.
+---
+
+## Introduction
+
+Ballerina is a programming language that simplifies integration by providing a large library of pre-built connectors. These connectors are special packages consisting of one or more [Ballerina clients](https://ballerina.io/learn/by-example/client-class/), which allow communication with external services, usually via REST APIs. By using connectors, developers can quickly integrate third-party services into their Ballerina applications without having to worry about the technical details of API interactions.
+
+Along with its powerful library ecosystem, Ballerina also allows developers to easily create, share, and manage client connectors. These connectors are typically published on Ballerina Central as public connectors, available to the entire community, but you can also publish them as private connectors for internal use within your organization.
+
+In this guide, we'll walk you through how to generate your first Ballerina connector using an OpenAPI specification. This is one of the fastest and easiest ways to build connectors, enabling you to quickly integrate external services into your Ballerina projects.
+
+## Prerequisites
+
+Before we begin, make sure you have:
+
+1. A basic understanding of Ballerina Swan Lake and the [latest version installed](https://ballerina.io/downloads/).
+2. An OpenAPI specification of the API for which you’re building the connector, along with the relevant API credentials where required (e.g., Twitter Developer account keys).
+3. A GitHub account, and Git installed locally.
+4. Visual Studio Code with the [Ballerina extension](https://marketplace.visualstudio.com/items?itemName=WSO2.ballerina) installed.
+
+## Step 1: Set the project structure
+
+> **Important**:If you'd like to contribute your project as an official Ballerina connector, you can skip the steps below and, reach out to the Ballerina team to create a new repository (under [`ballerina-platform`](https://github.com/ballerina-platform/) organization) and to set up the initial project structure. 
```

Comment:
Few points:
- Missing space after `Important:`
- `Reach out to Ballerina team` is not good from community POV. At least mention how to reach? Via discord?

Reply:
+1. Updated in https://github.com/ballerina-platform/ballerina-dev-website/pull/9507.